rust/main/agents/relayer/src/relayer.rs

```diff
@@ -135,12 +135,10 @@ impl BaseAgent for Relayer {
             .map(|origin| (origin.clone(), HyperlaneRocksDB::new(origin, db.clone())))
             .collect::<HashMap<_, _>>();
 
-        let mailboxes = settings
-            .build_mailboxes(settings.destination_chains.iter(), &core_metrics)
```

Comment:
are these used anymore? https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/main/hyperlane-base/src/settings/base.rs#L148-L152

if not just delete imo

Reply:
Still being used :+1: